### PR TITLE
Make ILB count configurable and increase default to 2.

### PR DESCRIPTION
**Pull request by prameshj**
**What type of PR is this?**

**What this PR does / why we need it**:
This PR makes the number of ILB services in the scale test to be configurable.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by prameshj** [comment]
Eventually all tests will start using the template being added in https://github.com/kubernetes/perf-tests/pull/1785

Until we maintain the current config file, it will be useful to configure the ILBs being deployed.

cc @freehan 

/assign @tosi3k @jkaniuk

**Review by tosi3k — CHANGES_REQUESTED** [review]
One minor remark, other than that LGTM.

File: clusterloader2/testing/l4ilb/config.yaml [hunk 1]
```diff
@@ -5,6 +5,9 @@
 {{$LARGE_BACKEND_LABEL := "ilb-large"}}
 {{$MEDIUM_BACKEND_LABEL := "ilb-medium"}}
 {{$SMALL_BACKEND_LABEL := "ilb-small"}}
+{{$LARGE_BACKEND_LB_SERVICE_COUNT := DefaultParam .LARGE_BACKEND_LB_SERVICE_COUNT 2}}
```
[the remark this answers is not in the document]
Comment (prameshj) [reply clusterloader2/testing/l4ilb/config.yaml]: Done. Thanks!

**Pull request merged**